### ISSUE TITLE
Having problem with linking the code together.

### ISSUE THREAD

**Issue by abdullahzahid10** [endrizzimarco/FreshFarm#9]
I created a new tab on the sidebar called 'FarmersDashboard' but the issue im having is that the code is not linking to the FarmersDashboard.vue instead it is navigating to the homepage under the farmers-dashboard.vue url

**Comment by georgi-ivanov19** [comment]
Firstly, it would be useful if you provide code snippets when raising an issue, so we can spot any mistakes with the code.

However, assuming your code is okay, it sounds to me that you are missing a route to the `FarmersDashboard.vue` component.
Routes are defined in `frontend/src/router/routes.js`. 

https://router.vuejs.org - here is the documentation for the vue router. 

Let me know if there is any other issues.

**Comment by georgi-ivanov19** [comment]
Okay I have looked at your branch, and you seem to have the route defined and are linking to it. I will try and replicate the issue and get back to you.

**Comment by endrizzimarco** [comment]
I have pushed the fix. Just pull from vscode and it should work. 

The issue was caused by the route being in the wrong place (see image below, should be a child of '/'). There was also an issue where it would open the dashboard in a new page; every link has been changed to `$router.push(route)` to route internally.

<img width="688" alt="image" src="https://user-images.githubusercontent.com/60980771/227727548-b3a9e52e-78b0-420d-bbf8-0425387d8e4c.png">
